tests/console/docker_runc.pm

```diff
@@ -34,20 +34,6 @@ sub run {
 
     # Setup the required testing environment
 
-    # install the docker package if it's not already installed
```

Comment:
Then please also delete the comment in L35 and most likely also the record_info in L33. But don't we need to start the docker service still?

Reply:
It is all done in function **install_docker_when_needed**